src/markdown/tutorial/part-2/11-ember-data.md

```diff
@@ -251,60 +253,97 @@ Darn, there were a couple of failing tests! At the same time, it's great that we
+
+The next step that we need to do, is to configure `RequestManager` to use this handler. Let's create the request-manager service.
 
-Inside this newly created file, we defined an `ApplicationAdapter` class, inheriting from the built-in [`JSONAPIAdapter`](https://api.emberjs.com/ember-data/release/classes/JSONAPIAdapter). This allows us to inherit all the default JSON:API functionalities, while customizing the things that didn't work for us by default. Specifically:
+```run:file:create cwd=super-rentals filename=app/services/request-manager.js
+import RequestManager from '@ember-data/request';
+import Fetch from '@ember-data/request/fetch';
+import { JsonSuffixHandler } from 'super-rentals/utils/handlers';
 
-* Our resource URLs have an extra `/api` *namespace* prefix.
-* Our resource URLs have a `.json` extension at the end.
+export default class extends RequestManager {
+  constructor(args) {
+    super(args);
 
-Adding a namespace prefix happens to be pretty common across Ember apps, so the `JSONAPIAdapter` has an API to do just that. All we need to do is to set the  `namespace` property to the prefix we want, which is `api` in our case.
+    this.use([JsonSuffixHandler, Fetch]);
+  }
+}
+```
+
+Notice that we are using the `JsonSuffixHandler` we created earlier. We also use the `Fetch` handler, which is a built-in handler that makes the actual fetch request. The `use` method is used to add handlers to the request manager. The order in which handlers are added is important, as they will be executed in the order they were added.
 
-Adding the `.json` extension is a bit less common, and doesn't have a declarative configuration API of its own. Instead, we will need to *[override][TODO: link to override]* EmberData's [`buildURL`](https://api.emberjs.com/ember-data/release/classes/JSONAPIAdapter/methods/buildURL?anchor=buildURL) method. Inside of `buildURL`, we will call `super.buildURL(...args)` to invoke the `JSONAPIAdapter` default implementation of `buildURL`. This will give us the URL that the adapter *would have built*, which would be something like `/api/rentals` and `/api/rentals/grand-old-mansion` after configuring the `namespace` above. All we have to do is to append `.json` to this URL and return it.
+Lastly, let's update our `store` service to use the new `RequestManager` we created.
 
-Similarly, serializers are located at `app/serializers`. Adapters and serializers are always added together as a pair. We added an `application` adapter, so we also added a corresponding serializer to go with it as well. Since the JSON data returned by our server is JSON:API-compliant, the default [`JSONAPISerializer`](https://api.emberjs.com/ember-data/release/classes/JSONAPISerializer) work just fine for us without further customization.
+```run:file:patch lang=js cwd=super-rentals filename=app/services/store.js
+@@ -1 +1,7 @@
+-export { default } from 'ember-data/store';
++// eslint-disable-next-line ember/use-ember-data-rfc-395-imports
```

Comment:
question: Is this eslint disable directive required?

Is there an issue tracking to fix this, as that seems unintuitive?

Reply:
Hey great question

So the reason I left it there, is as there is a lint rule, that would error if you extend store from `ember-data/store`. It is a good rule if your app is using new APIs everywhere.
But `ember-data` is still in the default output for `ember new` apps, so it's a bit conflicting. To use new APIs you need to create a store service, and add a custom request-manager... but at the same time if you just want to use `ember-data/store` you would trip off this rule. I kinda had 2 options:

1. Add is eslint ignore and just show how to do new things without changing packages 

2. Go update package json, install `@ember-data/store`, `@ember-data/request-utils`, `@ember-data/json-api`, and configure service out of new packages... We have plans on an interactive CLI installer for that things... it's just not there yet. Don't want to overwhelm new users

We will discuss it on today's EData call, thanks for your time @IgnaceMaes

Reply:
Alright, thanks for explaining.

I'm all for incremental improvements so let's go forward like this.